### DISCUSSION TITLE
Difficulty installing Liewa

### DISCUSSION THREAD

**Discussion by Tristanruel** [lennart-rth/Live-Earth-Wallpapers#33] [Q&A]
Hey there,

I'm trying to install Liewa from the Release pages for Windows, and I'm having quite a bit of trouble. I was able to configure a config.yml file in the GUI, however, I've had no luck with running the program. I've tried running the scheduler on GOES-16 and to no avail, have I even able to get any sort of image or action. After trying the test in the scheduler, the GUI crashes... anyone have any tips or experience with this?

Thank you in advance!

**Comment by lennart-rth** [comment]
Hi Tristan,
apologize for the inconvenience. Currently, I don't have access to a Windows machine, making it difficult for me to identify the issue without more information. 
Which version of Windows?
Could you provide any (error-)logs? 
Can you share the command line content when launching the app from there?
The software creates a Task in the Task Scheduler with the name "liewa." Is there an error log or any log messages available?

What I guess is that the problem may be related to updating the background image. Operating systems generally restrict software from modifying system settings like the background image, making it challenging to find a universal and reliable solution.
I have not found one yet, but if you happen to know or have developed one, feel free to make a Pull request.

About the scheduler:
The code responsible for setting the background image on Windows can be found in `liewa/liewa_cli/set_background.py.` There should be an alternative method for setting the background image, which is currently commented out. You could try enabling that method and build the project yourself.
Alternatively, you could write your own code in this file to set the background image. (Ther will definitly be a way to do this for your mashine:)

About the test crashing:
When trying the test in the scheduler, the app uses a visual Basic Script (cli.vbs)  to execute the cli.bat that in turn launches cli.py which starts the downloading. However the Background will not be set by this. It only updates the image "liewa/liewa_cli/recourses/backgroundImage.png". the crash is probably due to vbs. Maybe you can try to execute the script locally ans se what the error says?
